config/environments/production.rb

```diff
@@ -10,7 +10,7 @@
   config.action_controller.perform_caching = true
 
   # Disable Rails's static asset server (Apache or nginx will already do this)
-  config.serve_static_assets = false
+  config.serve_static_files = false
```

Comment:
We need to make sure this change is also made in the private_config versions of the environment files.